docs/debugging.md

```diff
@@ -5,7 +5,24 @@ Terms used:
 - *host*: The system that will have all devices connected to it and
     will receive the EC logs
 
-## Parallel port
+## Debugging with target device
+
+[log_cflag]: https://github.com/system76/ec/blob/master/src/board/system76/common/common.mk#L31-L39
```

Comment:
- Move the link to the end of the section
- Use a permalink. This way it will remain valid even if the file changes.
  - Press `y` on the page if you have keyboard shortcuts
  - Or just replace `master` with a commit hash, e.g.: `https://github.com/system76/ec/blob/01907011bb63/src/board/system76/common/common.mk#L31-L39`